src/main/java/io/prestosql/plugin/tiledb/TileDBModule.java

```diff
@@ -204,6 +220,8 @@ else if (type.equals(DATE)) {
                 }
                 return new ArrayList<Integer>();
             }
+            case TILEDB_DATETIME_MS:
```

Comment:
Do you need to support other DATETIMES here? Where is `getJavaListForType` used?

Reply:
It looks like that this is not used at the moment, there is no caller function in the codebase